rp_asos/migrations/0009_screening_record_date_nullable.py

```diff
@@ -0,0 +1,16 @@
+# Generated by Django 2.1.7 on 2019-05-13 06:27
```

Comment:
I think, if possible, we should combine these two migrations in to a single migration